StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/Link/PaymentDetails.swift

```diff
@@ -128,9 +92,8 @@ extension ConsumerPaymentDetails {
 
 extension ConsumerPaymentDetails.Details {
     /// For internal SDK use only
-    @objc(STP_Internal_ConsumerPaymentDetails_CardChecks)
-    class CardChecks: NSObject, STPAPIResponseDecodable {
-        enum State: String {
+    final class CardChecks: Codable {
+        enum CVCCheck: String, Codable {
```

Comment:
I might recommend using `SafeEnumCodable` for some of these. That way, a new enum case will fall through to the `.unparsable` case instead of causing the entire struct to fail decoding.

Reply:
TIL! That's an excellent call, I'll use that here. Does that require making the parent objects `UnknownFieldsCodable`?

Reply:
It shouldn't be needed: `UnknownFieldsCodable` is an optional way of accessing extra fields from the raw JSON at runtime. It's somewhat documented in StripeCodable.swift, but I'll write up some more complete docs with examples.